sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/administration/ServiceBusAdministrationAsyncClient.java

```diff
@@ -2562,17 +2562,34 @@ private static Throwable mapException(Throwable exception) {
 
```

Comment:
Do you have a live playback record for this?

Reply:
added play back.